File: src/com/turlington/Notify.java
package com.turlington;

/**
 * For testing right now.
 * Created by Mitchell on 7/18/2016.
 */
public class Notify implements iNotify {
	private final String fromEmail, password, toEmail;


	public Notify(String fromEmail, String password, String toEmail) {
		this.fromEmail = fromEmail;
		this.password = password;
		this.toEmail = toEmail;
	}

    @Override
    public void announce(JobListing jobListing) {
        EmailUtil.sendEmail(fromEmail, password, toEmail, jobListing);
    }
}

File: src/main/kotlin/com/turlington/Notify.kt
package com.turlington

/**
 * For testing right now.
 * Created by Valerie on 7/18/2016.
 */
class Notify internal constructor(private val emailInfo: EmailInfo) : iNotify {
    override fun announce(jobListing: JobListing) = EmailUtil.sendEmail(emailInfo, jobListing)
}
